backend/app/assets/javascripts/spree/backend/admin.js

```diff
@@ -58,6 +58,14 @@ Spree.ready(function(){
   $('.spree_add_fields').click(function() {
     var target = $(this).data("target");
     var new_table_row = $(target + ' tr:visible:last').clone();
+
+    // id of table row
+    new_table_row.attr("id", "spree_new_"+new_table_row.data("hook"));
+
+    // sort handle
+    new_table_row.find("td").first().empty();
+
+    // form ids
```

Comment:
I can't recall what the following code does to form ids, if we add a comment can we try to explain that, `form ids` is not really helping to understand.

Reply:
I assume a unique id is given to make sure it does not interfere with given ids of persistent items. Its needed to be created when saving new items. Maybe comment it with "set unique form ids for new items"?